crates/ark/src/variables/variable.rs

```diff
@@ -1336,6 +1433,88 @@ impl PositronVariable {
 
         Ok(out)
     }
+
+    fn try_inspect_custom_method(value: SEXP) -> Result<Option<Vec<Variable>>, harp::Error> {
+        let result: Option<RObject> = ArkGenerics::VariableGetChildren
+            .try_dispatch(value, vec![])
+            .map_err(|err| harp::Error::Anyhow(err))?;
+
+        match result {
+            None => Ok(None),
+            Some(value) => {
+                // Make sure value is a list before using inspect_list
+                if !r_typeof(value.sexp) == LISTSXP {
+                    return Err(harp::Error::Anyhow(anyhow!(
```

Comment:
Should this method return an `anyhow::Result` instead of `harp::Result`?

Reply:
Yes! https://github.com/posit-dev/ark/pull/561/commits/98f9996d47ccc65882b97f89ea6afdcecf303724